### ISSUE TITLE
Error building qt GUI on Ubuntu

### ISSUE THREAD

**Issue by serhannn** [andrewssobral/bgslibrary#56]
Hi, I'm on Ubuntu 16.04 with GCC version 5.4.1. While making the qt gui, I get the following error related to `libgtk`:

```
[  2%] Automatic moc and uic for target libbgs
[  2%] Automatic moc and uic for target bgslibrary_gui
[  2%] Built target bgslibrary_gui_automoc
[  2%] Built target libbgs_automoc
[ 93%] Built target libbgs
[ 94%] Linking CXX executable bgslibrary_gui
//usr/lib/x86_64-linux-gnu/libgtk-3.so.0: undefined reference to `cairo_surface_set_device_scale'
//usr/lib/x86_64-linux-gnu/libgtk-3.so.0: undefined reference to `cairo_surface_get_device_scale'
collect2: error: ld returned 1 exit status
CMakeFiles/bgslibrary_gui.dir/build.make:280: recipe for target 'bgslibrary_gui' failed
make[2]: *** [bgslibrary_gui] Error 1
CMakeFiles/Makefile2:68: recipe for target 'CMakeFiles/bgslibrary_gui.dir/all' failed
make[1]: *** [CMakeFiles/bgslibrary_gui.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```

I did a Google search and found out that some people got rid of this error by removing `anaconda/lib` folder from their `LD_LIBRARY_PATH`. I tried that but it did not solve the problem for me. Any suggestions?

**Comment by andrewssobral** [comment]
Hi @serhannn thank you for reporting this issue, I am checking this...

**Comment by serhannn** [comment]
@andrewssobral Thank you, but never mind, I solved the problem and got it working. It was an issue about the linkage of `libgtk-3`. Through the command `ldd /usr/lib/x86_64-linux-gnu/libgtk-3.so.0`, I figured out that it depends on `libcairo` which was pointing to the `lib` directory in my Anaconda installation. I found out that the library was outdated and update it to version 1.14.8 by `conda update cairo`. This solved my problem.
